proto/regen/ecocredit/v1/tx.proto

```diff
@@ -576,4 +599,72 @@ message MsgBridgeReceiveResponse {
   // project_id is the unique identifier of the project that was either created
   // or the existing project within which the credit batch exists.
   string project_id = 2;
-}
\ No newline at end of file
+}
+
+// MsgAddClassCreators is the Msg/AddClassCreators request type.
+// Since Revision 1
+message MsgAddClassCreators {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1;
+
+  // creators is the list of addresses to add to the class creator list.
+  repeated string creators = 2;
+}
```

Comment:
Similar to adding an allowed denom, we should only support adding a single class creator at a time. It is possible to submit a proposal with multiple messages adding multiple class creators (or other messages that update other params) but this is a big decision and we want to encourage the community to submit one addition at a time.